lib/Doctrine/Migrations/Tools/Console/Command/DoctrineCommand.php

```diff
@@ -105,8 +104,12 @@ protected function canExecute(
         return ! $input->isInteractive() || $this->askConfirmation($question, $input, $output);
     }
 
-    protected function procOpen(string $editorCommand, string $path) : void
+    protected function procOpen(OutputInterface $output, string $editorCommand, string $path) : void
     {
-        proc_open($editorCommand . ' ' . escapeshellarg($path), [], $pipes);
+        $process = new Process([$editorCommand, $path]);
+        $process->setTty(true);
```

Comment:
I guess this must be why Appveyor is failing

Reply:
Do you know if the feature of opening an editor works/worked at all on windows systems?

(or was simply never tested...)

If so, can we simply skip the test?

Reply:
No. (no). Probably, I don't know why an editor is involved here TBH.

Reply:
What about dropping the possibility of opening an editor from doctrine migrations? I think that is out of scope for such library...

Reply:
Had no idea this was a thing… what is it used for?

Reply:
When you run the diff or generate command, and pass `--editor-cmd=vi`, it will open the generated migration file with vi

Reply:
Oh ok… well I know I wouldn't use it, because I don't like having many instances of vim running for the same project. In favor of dropping that feature.

Reply:
@alcaeus what do you think?

Reply:
I've never used it either. If it's causing problems, drop it. If it's easier to skip something on Windows, do that. I don't have any preference here ;)

Reply:
I will drop it then, as it will allow us to not depend on the `symfony/process` component.

Reply:
done, removed